src/Ng/packages.config

```diff
@@ -24,10 +24,10 @@
   <package id="Microsoft.Net.Http" version="2.2.22" targetFramework="net452" />
   <package id="Microsoft.Web.Xdt" version="2.1.1" targetFramework="net452" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="3.1.0" targetFramework="net452" />
-  <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net452" />
+  <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
   <package id="NuGet.Common" version="3.5.0-beta2-1484" targetFramework="net452" />
   <package id="NuGet.Core" version="2.8.5" targetFramework="net452" />
-  <package id="NuGet.Services.KeyVault" version="1.0.0.0" targetFramework="net452" />
+  <package id="NuGet.Services.KeyVault" version="1.0.14-r-scottbom-keyrot" targetFramework="net452" />
```

Comment:
Do we really want to depend on this prerelease package right now? Are we getting close to doing an updated release of this?